firmware/src/hal/sgx/test/mock/mock_seal.c

```diff
@@ -128,8 +128,8 @@ void assert_oe_seal_called_with(const void* plugin_id,
                                 size_t additional_data_size) {
     assert(G_oe_seal_args.plugin_id == plugin_id &&
```

Comment:
I've just realised the existence of this assert of multiple conditions joined by `&&`. We should rewrite this as multiple `assert`s to aid in debugging.